descriptors/ModuleDescriptor-template.json

```diff
@@ -31,6 +31,11 @@
           "methods": ["DELETE"],
           "pathPattern": "/remote-storage/configurations/{configurationId}",
           "permissionsRequired" : [ "remote-storage.configurations.item.delete" ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/remote-storage/providers",
+          "permissionsRequired" : [ "remote-storage.providers.get" ]
```

Comment:
@VitaliiZahoruiko , please, add remote-storage.providers.get to the permissions description section and add it to the remote-storage.all permission.

Reply:
Done. Also, permission renamed to remote-storage.providers.collection.get to be more consistent